src/GitTabExpansion.ps1

```diff
@@ -168,39 +168,43 @@ function script:gitTfsShelvesets($filter) {
         quoteStringWithSpecialChars
 }
 
-function script:gitFiles($filter, $files) {
+function script:gitFiles($filter, $gitDir, $files) {
+    [string] $repoDir = Split-Path -Path $gitDir -Parent
```

Comment:
Minor nit, the `[string]` cast is unnecessary since Split-Path returns a string.

Reply:
Fixed.